tools/pyutils/torchconverter.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+##
+# Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+#
+# @file transLayer_v2.py
```

Comment:
transLayer_v2.py?

Reply:
It reuse torchconverter.py in pyutils. thanks!